tck-test/TestInterpreter.h

```diff
@@ -27,10 +29,19 @@ class CancelCommand;
 class AssertCommand;
 
 class TestInterpreter {
+  class TestClient {
+   public:
+    TestClient(std::shared_ptr<RSocketClient> client)
+        : client_(std::move(client)) {
+      auto rs = client_->connect().get();
+      requester_ = std::move(rs);
+    }
+    std::shared_ptr<RSocketClient> client_;
+    std::shared_ptr<RSocketRequester> requester_;
```

Comment:
nit. public member without underscore.  OR it can be a private member with underscore (and provide an accessor).